src/components/App.jsx

```diff
@@ -7,32 +7,60 @@ import NewButton from './NewButton';
 //React Hooks
 import { useEffect, useState } from "react";
 
+let LOCALHOST = "localhost";
```

Comment:
Would be better to create a variable with the whole API prefix instead. This will be useful soon when we deploy this to the web.

```suggestion
let API_SERVER_PREFIX = "http://localhost:4000";
```

And replace all usage from something like this `http://${LOCALHOST}:4000/tags` to this `{API_SERVER_PREFIX }:4000/tags`